defaults/main.yml

```diff
@@ -43,6 +43,10 @@ nim_waku_p2p_max_connections: 150
 # maximum number of stored historical messages
 nim_waku_store_capacity: 10000
 
+# SQLite store
+nim_waku_sqlite_store: false
+nim_waku_sqlite_retention_time: 30.days.seconds
```

Comment:
Could we explain a bit this weird duration format? Also, this text should be single-quoted.

Maybe we could also start with a lower default in the role? 7 days?

Reply:
I think this is simply an "easy" way to define "30 days worth of seconds" in chronos. Agreed re lowering the default here.

Reply:
The idea was to allow a chronos duration as type for the parameter. But it is not supported out of the box, so I went with seconds for now.